# Q5

A motor with 16 kW power is attached to a compressor via a flat belt with a length of 3 m.
The belt experienced a maximum stress of 4 MPa in an operation and medium shock with
Ks =1.5. If the coefficient of friction between the belt surface and the pulleys are 0.4 and the density of the belt is 950 kg/m3 and the details of the belt drive are tabulated in Table 1. Determine;

Table 1

|  | Diameter | Speed |
|:--------:|:--------:|:--------:|
|  Compressor Pulley   |  0.6   |  1200   |
|  Motor Pulley   |  dm   |  1800   |

i)	the diameter of the motor pulley, dm

ii)	the center distance, c between the pulleys.

iii)angle of wrap/ contact angle, φ of both pulleys. Does the angle equal? In what circumstances the angles will be equal?

iv)	torque on both pulleys. Does the torque on both pulleys equal? Why? In what circumstances the torque will be equal? How about power?

v)	Velocity of the belt, v

vi)	Centrifugal force, Fc

vii)	Cross section of belt, A

viii)	Tight tension on belt, F1

ix)	Slack tension on belt, F2

x)	The initial tension on the belt, Fi



In [45]:
from math import pi, sin, cos, sqrt, pow, radians, degrees, asin, e

In [46]:
motorPower = 16000
print('Belt type: Flat Belt')
beltLength = 3
beltStress = 4E+6
Ks = 1.5
friction = 0.4
beltDensity = 950
driver_rpm = 1800
driven_rpm = 1200

Belt type: Flat Belt


i)	the diameter of the motor pulley, dm

Use speed ratio equation

$$n_1r_1=n_2r_2$$
or
$$n_1d_1=n_2d_2$$
$$(1800)d_m=(1200)(0.6)$$


In [47]:
dm=1200*0.6/1800
print(f'dm={dm}m')

dm=0.4m


ii)	the center distance, c between the pulleys.

$$c = \frac{1}{4}\left( b+ \sqrt {b^2-8((r_2+r_1)^2} \right)$$

Where b

$$b=L-\pi (r_2-r_1)^2$$

In [48]:
r1 = 0.2
r2 = 0.3
b=round(beltLength-(pi*(r2+r1)),4)
print(b)
c=0.25*(b+sqrt(pow(b,2)-(8*pow(r2-r1,2))))
print(f'Center distance, c ={c:.3f}m')

1.4292
Center distance, c =0.708m


iii)angle of wrap/ contact angle, $\alpha$ of both pulleys. Does the angle equal? In what circumstances the angles will be equal?

$$sin\alpha = \frac{r_2-r_1}{c}$$

In [49]:
c = round(c,3)
alpha = asin((r2-r1)/c)
print(f'alpha={alpha}')
print(f'alpha in degrees = {degrees(alpha)}')
smallPulley = pi-(2*alpha)
largePulley = pi+(2*alpha)
print(f'Small pulley, phi={degrees(smallPulley)}')
print(f'Large pulley, phi={degrees(largePulley)}')
print('Not equal')
print('Equal when alpha=0')

alpha=0.14171682713853
alpha in degrees = 8.119776081022817
Small pulley, phi=163.76044783795436
Large pulley, phi=196.23955216204564
Not equal
Equal when alpha=0


iv)	torque on both pulleys. Does the torque on both pulleys equal? Why? In what circumstances the torque will be equal? How about power?

Use Power equation
$$P=\omega T$$
$$P=\frac {2\pi n_{rpm}}{60}T$$

In [50]:
T_driver=motorPower*60/(2*pi*driver_rpm)
print(f'Small Pulley, T={T_driver}Nm')
T_driven=motorPower*60/(2*pi*driven_rpm)
print(f'Large Pulley, T={T_driven}Nm')

print('Torque not equal. The purpose of transfer the rotation to larger pulley to generate torque')
print('Torque is same when radius is same')
print('In ideal condition, power is transferred 100%. This ideal condition is efficiency 100%')

Small Pulley, T=84.88263631567752Nm
Large Pulley, T=127.32395447351628Nm
Torque not equal. The purpose of transfer the rotation to larger pulley to generate torque
Torque is same when radius is same
In ideal condition, power is transferred 100%. This ideal condition is efficiency 100%


v)	Velocity of the belt, v

$$V = \frac {\pi dn_{rpm}}{60}$$


In [51]:
velocity = pi*dm*driver_rpm/60
print(f'Velocity, V={velocity:.3f}m/s')

Velocity, V=37.699m/s


vi)	Centrifugal force, Fc

$$F_c = \rho A V^2$$

In [52]:
velocity = round(velocity,2)
fc = beltDensity*pow(velocity,2)
print(f'Fc ={fc:.1f}A')
print('Left Fc with unknown A')

Fc =1350225.5A
Left Fc with unknown A


vii)	Cross section of belt, A

Related equations
$$\sigma = \frac{F_{max}}{A}$$
$$\sigma = \frac{K_s F_1}{A}$$
$$4MPa = \frac{1.5\times F_1}{A}$$

In [53]:
print(f'F1={4E+6/1.5}A')

F1=2666666.6666666665A


Use
$$F_1=\frac{4\times10^6}{1.5}A$$
as equation 1

Use $F_1$ equation
$$F_1=F_c+\left[ \frac{\gamma}{\gamma-1} \right]\frac{T_1}{r_1}$$
where
$$\gamma =e^\frac{f \phi}{sin \beta}$$
For flat belt, $\beta=\frac{180}{2}=90$

$\phi$ in degree

In [54]:
smallPulley = round(smallPulley,3)
print(f'phi = {smallPulley}')
gamma = pow(e,friction*smallPulley/sin(radians(90)))
print(f'gamma={gamma}')

phi = 2.858
gamma=3.1367900500987753


Substitute $\gamma$ to F1
$$F_1=1350225.5A+\left[ \frac{3.14}{3.14-1} \right]\frac{84.88}{0.2}$$

In [55]:
T_driver=84.88
rightSide = (3.14/(3.14-1))*T_driver/r1
print(f'F1 = {fc:.1f}A+{round(rightSide,2)}')

F1 = 1350225.5A+622.72


Use
$$F_1=1350225.5A+622.72$$
 as equation 2

Make equation 1 and 2 equal
$$\frac{4\times10^6}{1.5}A=1350225.5A+622.72$$
$$\frac{4\times10^6}{1.5}A-1350225.5A=622.72$$

In [56]:
Area = round(rightSide,2)/((beltStress/Ks)-fc)
print(f'A={Area:.2e}m^2')

A=4.73e-04m^2


viii)	Tight tension on belt, F1

Use
$$F_1=1350225.5A+622.72$$

In [57]:
F1 = round(fc*Area+rightSide,2)
print(f'F1={F1}N')

F1=1261.42N


ix)	Slack tension on belt, F2

Use equation for flat belt

$$\frac{F_1-F_c}{F_2-F_c}=e^{f \phi}$$
$$\frac{1261.42-F_c}{F_2-F_c}=e^{0.4(2.858)}$$

From previous equation
$$F_c=\rho A V^2$$
$$F_c=1350225.5A$$

In [58]:
Area = 4.73E-4
fc = round(1350225.5*Area,2)
print(f'Fc={fc}N')

Fc=638.66N


Therefore
$$\frac{1261.42-638.66}{F_2-638.66}=e^{0.4(2.858)}$$

In [67]:
1261.42-638.66

622.7600000000001

In [59]:
F2 = round(((F1-fc)/pow(e,friction*smallPulley))+fc,2)
print(f'F2={F2}N')

F2=837.19N


x)	The initial tension on the belt, Fi

$$F_i=\frac{F_1+F_2}{2}

In [60]:
Fi = (F1+F2)/2
print(f'Initial tension, Fi ={Fi}N')

Initial tension, Fi =1049.305N


Well-defined questions

1.	If the area of the belt is reduced to A=0.0001m2, suggest the number of belts required using the same tensions in Question 1.

2.	Compare the power capacity of the belt drive if the flat belt is replaced by another belt as detailed below: 

Table 1

| Belt Types | Specific Weight (kg/m^3) | Allowable Tensile Strength (MPa) | CoF | Dimension|Included angle|
|:--------:|:--------:|:--------:|:--------:|:--------:|:--------:|
|  V-belt   |  950   |  4   | 0.4 |A=2000mm^3 |38|
|  Round belt   |  950   |  4   | 0.4 |d=2mm |-|

3.	  Compare the value of the belt area in Question 1 if the speed of the motor is reduced to 20%. How about the belt numbers? 

In [61]:
print(f'F1={F1}')
print('New A = 0.0001 m^2')
Area = 0.0001

F1=1261.42
New A = 0.0001 m^2


$$\sigma=\frac{F_{max}}{A_{new}}$$
$$4MPa=\frac{F_{max}}{0.0001}$$

In [62]:
Fmax=beltStress*Area
print(f'Fmax = {Fmax}N')

Fmax = 400.0N


Fmax for 1 belt only support 400N. Current F1 must able to sustain load of 1261.42N.

number of belt X 400 =1261.42

or the equation will be

$$n_{belt}\times F_{max}=F_1$$

In [63]:
nBelt = F1/Fmax
print(f'n={nBelt}')
print('Number of belt is 4')

n=3.15355
Number of belt is 4


2. Compare the power capacity of the belt drive if the flat belt is replaced by another belt

Equation that can be used


$$\frac{F_1-F_c}{F_2-F_c}=e^{\frac{f \phi}{\sin \beta}}$$
$$\sigma = \frac{F_{max}}{A}$$

$$F_c=\rho AV^2$$

Get $F_1$ and $F_2$

then substitute into
$$P=(F_1-F_2)V$$


In [64]:
print('V-belt')
print(f'phi=2.858')
print(f'beta = 19')


V-belt
phi=2.858
beta = 19


$$\sigma = \frac{F_{max}}{A}$$
$$4\times10^6 = \frac{F_{max}}{A}$$

In [65]:
beltDensity=950
Area = 2000E-6
F1=beltStress*Area
print(f'F1={F1}')

F1=8000.0


$$F_c=\rho AV^2$$
$$F_c=(950)(2000\times10^{-6})(37.7)^2$$

In [66]:
Fc=beltDensity*Area*pow(velocity,2)
print(f'Fc={Fc}')

Fc=2700.4510000000005


Round belt

In [68]:
d_round_belt = 0.002
area_round_belt = pi*pow(d_round_belt,2)/4
print(f'Area of round belt, A={area_round_belt}m^2')

Area of round belt, A=3.141592653589793e-06m^2
